### PR TITLE
issue: 4007026 Disaster recovery pipeline

### PR DESCRIPTION
**Pull request by NirWolfer**
## Description
Support for release process in DRP is required, since the envrironment is different than the prod environment there should be a new pipeline defined with said changes

##### What
Add new job template definition to release_jjb
Add new dr matrix with configuration for PDC env

##### Why ?
Issue: HPCINFRA-1889

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

File: .ci/pipeline/release_jjb.yaml [hunk 1]
```diff
@@ -55,11 +55,72 @@
                 browser: githubweb
                 browser-url: "{jjb_git}"
         script-path: ".ci/Jenkinsfile"
+- job-template:
```
Comment (dpressle) [inline .ci/pipeline/release_jjb.yaml:58]: disabled: true ?

File: .ci/pipeline/release_matrix_job_dr.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  MAIL_FROM: jenkins@nvidia.com
+
+volumes:
+  # Default release location
+  - {mountPath: /auto/mswg/release/vma, hostPath: /auto/mswg/release/vma}
+  # User profile for release
+  - {mountPath: /var/home/swx-jenkins, hostPath: /labhome/swx-jenkins}
+
+runs_on_dockers:
+  - {file: '.ci/dockerfiles/Dockerfile.rhel8.6.release', name: 'rhel8.6', uri: '$arch/$name/release', build_args: '--no-cache', arch: 'x86_64', tag: '20240718'}
```
Comment (dpressle) [inline .ci/pipeline/release_matrix_job_dr.yaml:25]: pdc harbor url ?

File: .ci/pipeline/release_jjb.yaml [hunk 3]
```diff
@@ -55,11 +55,72 @@
...
+                browser: githubweb
+                browser-url: "{jjb_git}"
+        script-path: ".ci/Jenkinsfile"
 - project:
     name: libvma
     jjb_email: 'nwolfer@nvidia.com'
     jjb_proj: 'LibVMA-release'
     jjb_git: 'git@github.com:Mellanox/libvma.git'
     jjb_owner: 'Nir Wolfer'
+    jjb_folder: 'libvmna'
```
Comment (dpressle) [inline .ci/pipeline/release_jjb.yaml:121]: typo libvmna

**Review by dpressle — CHANGES_REQUESTED** [review]

File: .ci/pipeline/release_jjb.yaml (hunk 3 above)
Comment (NirWolfer) [reply .ci/pipeline/release_jjb.yaml:121]: fixed

File: .ci/pipeline/release_jjb.yaml (hunk 1 above)
Comment (NirWolfer) [reply .ci/pipeline/release_jjb.yaml:58]: added

File: .ci/pipeline/release_matrix_job_dr.yaml (hunk 2 above)
Comment (NirWolfer) [reply .ci/pipeline/release_matrix_job_dr.yaml:25]: added build arg to the image

**Review by dpressle — DISMISSED** [review]

**Comment by NirWolfer** [comment]
bot:restest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by dpressle** [comment]
https://nbupdc.blsm.nvidia.com/swx-media-master/job/libvma/job/LibVMA-release-dr/3/

**Comment by NirWolfer** [comment]
bot:lin:retest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by dpressle** [comment]
bot:retest

**Comment by dpressle** [comment]
bot:retest

**Comment by NirWolfer** [comment]
bot:retest

**Comment by dpressle** [comment]
bot:rm:retest

**Review by dpressle — APPROVED** [review]

**Pull request merged**